test/unit/core/tuple_utility_test.cpp

```diff
@@ -322,18 +320,20 @@ TYPED_TEST(tuple_utility, tuple_split_and_pop)
 {
     std::tuple t{float{2.1}};
     {
-        auto [left, right] = tuple_split<float>(t);
+        auto [left, right] = seqan3::tuple_split<float>(t);
 
         EXPECT_EQ(std::tuple_size_v<std::remove_reference_t<decltype(left)>>, 0u);
         EXPECT_EQ(std::tuple_size_v<std::remove_reference_t<decltype(right)>>, 1u);
 
-        using left_t = detail::transfer_template_args_onto_t<remove_cvref_t<decltype(left)>, type_list>;
-        using right_t = detail::transfer_template_args_onto_t<remove_cvref_t<decltype(tuple_pop_front(right))>, type_list>;
+        using left_t = seqan3::detail::transfer_template_args_onto_t<
+                            seqan3::remove_cvref_t<decltype(left)>, seqan3::type_list>;
+        using right_t = seqan3::detail::transfer_template_args_onto_t<
+                            seqan3::remove_cvref_t<decltype(seqan3::tuple_pop_front(right))>, seqan3::type_list>;
```

Comment:
same here

Reply:
If i would make the change as you wish, the 4th line would get too long:
```cpp
        using left_t = seqan3::detail::transfer_template_args_onto_t<seqan3::remove_cvref_t<decltype(left)>,
                                                                     seqan3::type_list>;
        using right_t = seqan3::detail::transfer_template_args_onto_t<seqan3::remove_cvref_t<decltype(seqan3::tuple_pop_front(right))>,
                                                                      seqan3::type_list>;
```